tests/step-112.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Test:
+#   Set up bare repos, run the install and verify the hooks get installed/uninstalled
+
+if echo "$EXTRA_INSTALL_ARGS" | grep -q "use-core-hookspath"; then
+    echo "Using core.hooksPath"
+    exit 249
+fi
+
+mkdir -p /tmp/test112
+cd /tmp/test112 && git init --bare || exit 1
+
+if grep -r 'github.com/rycus86/githooks' /tmp/test112/; then
+    echo "! Hooks were installed ahead of time"
+    exit 1
+fi
+
+mkdir -p ~/.githooks/templates
```

Comment:
If you do `mkdir -p ~/.githooks/templates/hooks` here, then this test already works on `master`.

Reply:
I know, but is this the behavior one wants?

Reply:
Yeah, I'm not sure.
Perhaps we should create that directory if `init.templateDir` is set?

Reply:
Yeah, definitely. Thas included in my fix. Wait I prepare the rebase changes...